src/main/java/dk/bec/polonez/reservationsystem/model/Reservation.java

```diff
@@ -33,4 +34,10 @@
     @ManyToOne
     @JoinColumn
     private Offer offer;
+
+    @Override
+    public String toString() {
+        // example info
```

Comment:
**advice** Minor: Doesn't comment seem redundant? And do you really need this ```toString()``` method?

Reply:
I need some kind of textual representation of reservation to be inserted in email. I changed its name and moved it to service.